FTBFS xmlto

```
2024/08/15 21:07:58 WARN xmlif/xmlif.l:46:8: error: type defaults to 'int' in declaration of 'ifsense' [-Wimplicit-int]
2024/08/15 21:07:58 WARN    46 | static ifsense;                 /* sense of last `if' or unless seen */
2024/08/15 21:07:58 WARN       |        ^~~~~~~
2024/08/15 21:07:58 WARN xmlif/xmlif.l:243:1: error: return type defaults to 'int' [-Wimplicit-int]
2024/08/15 21:07:58 WARN   243 | {
2024/08/15 21:07:58 WARN       | ^   
2024/08/15 21:07:58 WARN xmlif/xmlif.c:2002:16: warning: 'input' defined but not used [-Wunused-function]
2024/08/15 21:07:58 WARN  2002 |     static int input  (void)
2024/08/15 21:07:58 WARN       |                ^~~~~
2024/08/15 21:07:58 WARN xmlif/xmlif.c:1961:17: warning: 'yyunput' defined but not used [-Wunused-function]
2024/08/15 21:07:58 WARN  1961 |     static void yyunput (int c, register char * yy_bp )
2024/08/15 21:07:58 WARN       |                 ^~~~~~~
2024/08/15 21:07:58 WARN make[1]: *** [Makefile:741: xmlif/xmlif.o] Error 1
2024/08/15 21:07:58 INFO make[1]: Leaving directory '/home/build'
2024/08/15 21:07:58 INFO make: Leaving directory '/home/build'
2024/08/15 21:07:58 WARN make: *** [Makefile:584: all] Error 2
```

--- a/xmlto.yaml
+++ b/xmlto.yaml
@@ -48,6 +48,5 @@ subpackages:
     description: xmlto manpages
 
 update:
-  enabled: false
   enabled: true
   git: {}

--- a/xmlto.yaml
+++ b/xmlto.yaml
@@ -49,4 +49,5 @@ subpackages:
 
 update:
   enabled: false
-  exclude-reason: needs melange schema to support `git` update
+  enabled: true
+  git: {}

--- a/xmlto.yaml
+++ b/xmlto.yaml
@@ -1,7 +1,7 @@
 package:
   name: xmlto
-  version: 0.0.28
-  epoch: 4
+  version: 0.0.29
+  epoch: 0
   description: Convert xml to many other formats
   copyright:
     - license: GPL-2.0-or-later
@@ -27,10 +27,11 @@ environment:
       - perl-yaml-syck
 
 pipeline:
-  - uses: fetch
+  - uses: git-checkout
     with:
-      expected-sha256: 1130df3a7957eb9f6f0d29e4aa1c75732a7dfb6d639be013859b5c7ec5421276
-      uri: https://releases.pagure.org/xmlto/xmlto-${{package.version}}.tar.bz2
+      repository: https://pagure.io/xmlto.git
+      expected-commit: 635dcd5bc84a7b67a3a2050a179d96e1d3050b15
+      tag: ${{package.version}}
 
   - uses: autoconf/configure
 
@@ -47,6 +48,5 @@ subpackages:
     description: xmlto manpages
 
 update:
-  enabled: true
-  release-monitor:
-    identifier: 13307
+  enabled: false
+  exclude-reason: needs melange schema to support `git` update

